nautobot_capacity_metrics/metrics.py

```diff
@@ -120,6 +122,16 @@ def metric_models(params):
     yield gauge
 
 
+def metric_health_checks():
+    """Runs health checks and returns them in Prometheus Metric format."""
+    gauge = GaugeMetricFamily("nautobot_health_check", "Per Nautobot Model count", labels=["name"])
+    health_checker = CheckMixin()
+    health_checker.run_check()
```

Comment:
The Nautobot health checks are already super expensive, especially in a docker environment.  We need to be extra careful that we aren't running additional healthchecks but providing metrics on existing ones.  I suggest adding this in core nautobot is a wiser place for this.

Reply:
Understood, that makes sense. I will take a stab at implementing it in core.

Reply:
Closing in favor of https://github.com/nautobot/nautobot/pull/2759.